client/src/components/about1/AboutPage.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Content from "./Content";
+import { Link } from "react-scroll";
+
+function AboutPage() {
+  return (
+    <div>
+      <div 
+        className="hero min-h-screen"
+        style={{
+          backgroundImage:
+            "url(https://daisyui.com/images/stock/photo-1507358522600-9f71e620c44e.jpg)",
+        }}
```

Comment:
This background image looks out of context. It doesn't go well with our design 
Consider swapping it for something like the icons we used for the sidebar. Simple lines, simple colours, something that will complement the aesthetic we have.